user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+
+To avoid this step and improve security we strongly recommend to use the dedicated functionality described below.
+This section applies to both @gmail.com customers and Google Apps customers.(Check with your email administrator if you 
+doubt).
+
+- Select the Account Type - *"Gmail"* and click the :guilabel:`Connect` button that appears below.
+
+  |
+  
+  |EmailSync3|
+  
+  |
+  
+.. note::
+
+    The option is only available only if your OroCRM instance is connected with the Google Apps account. Please check 
+    with your system administrator if you want to enable Gmail synchronization for your account.
+
+- If you haven’t used the Google account in the browser (or its cache has been cleared), the Sign-in form will appear. 
+  Log-in to the account.
+
+   |
+  
+   |EmailSyncGM1|
+
+   |
+
+ - If you are using several Gmail accounts in the browser, you will see a list of them. Select the account for which you 
+   want to enable synchronization with OroCRM. Select the account and log-in (if necessary).  
+   
+   |
+   
+   |EmailSyncGM2|
+   
+   |
+   
+ - As soon as you have selected the account, or if you are already logged in into a single Gmail account, you need to 
+   allow OroCRM to view and manage your mail, and allow it offline access to your mailbox.
+   
+   |
+   
+   |EmailSyncGM3|
+   
+   |EmailSyncGM4|
+
+   |
+   
+.. hint::
+
+    Make sure that pop-up windows from your OroCRM instance are not blocked.
+
+
+- After the connection has been established, you will see the connected account name (your email address) and the list of 
+  folders that can be refreshed with the :guilabel:`Retrieve Folders` button.
+
+  |
+  
+  |EmailSyncGM5|
+  
+  |
+  
+Select the folders you want to synchronize and save your user profile. The synchronization will start immediately, but 
+the full sync can take a while depending on the size of your mailbox.
+
+Change the Synchronized Mailbox
+"""""""""""""""""""""""""""""""
+
+If you want to change the mailbox you sync with OroCRM, simply change the IMAP/SMTP credentials for generic IMAP sync, 
+or remove the connected Gmail account, change your primary email address, and sync with Gmail again.
+
+.. caution::
+
+    If you change the synced mailbox, all emails from the previous mailbox will be deleted from OroCRM.   
+    
+Configuring Mailboxes in the Multi-Organization Environment
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+If you want to work with email in :ref:`multiple organizations <user-guide-getting-started-change-organization>`, you 
+have to configure a mailbox for every organization. You can synchronize different external mailboxes or the 
+same external mailbox for your user in different organizations.
+
+
+Save the Changes
+----------------
+Click the :guilabel:`Save and Close` button in the top right corner to save the changes to your profile and return to 
+*"My User"* page.
```

Comment:
Edit:Click the :guilabel:`Save and Close` button in the top right corner to save the changes to your profile and return to the *"My User"* page.